test/e2e/reconciler/cache/replication_test.go

```diff
@@ -248,6 +248,9 @@ func TestAllScenariosAgainstStandaloneCacheServer(t *testing.T) {
 	err = clientcmd.WriteToFile(cacheServerKubeConfig, cacheKubeconfigPath)
 	require.NoError(t, err)
 
+	t.Logf("kubeconfig for the cache server is at: %v", cacheKubeconfigPath)
+	time.Sleep(10 * time.Hour)
```

Comment:
:rofl:

Reply:
yeah, i'm going to remove it before this PR merges :)